src/pages/keyword-generator/KeywordGenerator.vue

```diff
@@ -102,7 +104,10 @@ function onStopWordsToggle(checked) {
              <template #footer>
                <div class="ma-button-container">
                  <div class="ma-description-count">{{ inputInfo }}</div>
-                 <MaButton @click="clearAll" :size="medium" variant="stroke">Clear All</MaButton>
+                 <div class="ma-button-container-right">
+                    <MaButton @click="clearAll" :size="medium" variant="stroke">Clear All</MaButton>
+                    <MaButton @click="convert" :size="medium" variant="stroke" icon="arrow-right">Convert</MaButton>
```

Comment:
```suggestion
                    <MaButton :size="medium" variant="stroke" @click="clearAll">Clear All</MaButton>
                    <MaButton :size="medium" variant="stroke" icon="arrow-right" @click="convert">Convert</MaButton>
```

functional props can come at the end